mkdocs.yml

```
@@ -61,8 +61,9 @@ nav:
     - SAR FAQ: https://asf.alaska.edu/information/sar-information/what-is-sar/#sar_faq" target="_blank
   - Tutorials:
     - tutorials.md
+    - Using HyP3 Python SDK: https://nbviewer.jupyter.org/github/ASFHyP3/hyp3-sdk/blob/main/docs/sdk_example.ipynb
```

```suggestion
    - Using HyP3 Python SDK: https://nbviewer.jupyter.org/github/ASFHyP3/hyp3-sdk/blob/main/docs/sdk_example.ipynb" target="_blank
```
Adding the target=blank injection so it'll open in a new browser tab